### PR TITLE
Fix styling for landing page

### PR DESCRIPTION
**Pull request by ankeris**

### CODE REVIEW

File: src/app/@core/styles/global.scss [hunk 1]
```diff
@@ -33,6 +33,20 @@ h6 {
 .content-section {
     margin: 30px auto 100px auto;
     max-width: 1260px;
+    @media only screen and (max-width: 1260px) {
+        border: none;
+        margin: 0;
+    }
+}
+
+.content-section-landing {
```
Comment (ankeris) [inline src/app/@core/styles/global.scss:42]: should be 
```
.content-section {
    &__landing {}
}
```

And on the element use both classes - `content-section content-section__landing`
This way we have less css transpiled.

File: src/app/@modules/main-list/main-list.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <app-search-box (searchOnChange)="searchOnChange($event)"></app-search-box>
 
-<section class="content-section" *ngIf="searchRequest$ | async as searchRequest">
+<section class="content-section-landing" *ngIf="searchRequest$ | async as searchRequest">
```
Comment (ankeris) [inline src/app/@modules/main-list/main-list.component.html:3]: Add Element from BEM `content-section content-section__landing`.
Relates to the comment I left in CSS file.

File: src/app/@shared/near-me-items/near-me-items.component.html [hunk 3]
```diff
@@ -13,13 +13,13 @@ <h3 *ngIf="waitLocalVendors.waiting == false && waitLocalVendors.waitedForTooLon
...
         <ng-template #elseBlock>
-            <h3 *ngIf="!topVendorsNearMe.length && waitLocalVendors.waiting === false">{{ "NO_VENDORS_NEAR" | translate }}</h3>
+            <p class="noItemsText" *ngIf="!topVendorsNearMe.length && waitLocalVendors.waiting === false">😞 {{ "NO_VENDORS_NEAR" | translate }}</p>
         </ng-template>
         <!-- VENDORS FOUND IMMEDIATELY -->
         <app-list-vendors
             *ngIf="waitLocalVendors.waiting === false && waitLocalVendors.waitedForTooLong === false"
             [vendorsWithListings$]="topVendorsNearMe$"
-            [title]="'Vendors near your location'"
+            [title]="'📍 Vendors near your location'"
```
Comment (ankeris) [inline src/app/@shared/near-me-items/near-me-items.component.html:22]: <3 the emojis, maybe we should use Translation to this title:
```
[title]="VENDORS_NEAR_LOCATION | translate"
```
also add to translations.
I know it's not your implementation, but we might just fix it sooner :)

Comment (Vidassundk) [reply src/app/@shared/near-me-items/near-me-items.component.html:22]: I get a syntax error of that

File: src/app/@core/styles/global.scss [hunk 4]
```diff
@@ -77,10 +105,10 @@ p {
     font-family: "Roboto", sans-serif;
 }
 
-.layout {
-    @media only screen and (max-width: 640px) {
-        background-color: white !important;
-    }
+h1,
```
Comment (ankeris) [inline src/app/@core/styles/global.scss:108]: should change headings in nebular theme :)

File: src/app/@core/styles/global.scss [hunk 5]
```diff
@@ -33,18 +33,47 @@ h6 {
...
+        border-radius: 0;
+    }
+    &__vendor {
+        padding: 0;
+    }
+
+    &__product {
+        padding: 0;
+    }
+    &__landing {
```
Comment (ankeris) [inline src/app/@core/styles/global.scss:49]: clean up

File: src/app/@modules/topheader/topheader.component.html [hunk 6]
```diff
@@ -3,7 +3,10 @@
         <svg [routerLink]="globalRoutes.ROOT" class="top-header__button" height="50">
             <use xlink:href="assets/icons/pard-text.svg#Layer_1"></use>
         </svg>
+
         <img class="top-header__language" src="assets/images/flags/lithuania.svg" (click)="setLanguage('lt')" />
+
+        <!-- <img class="top-header__language" src="assets/images/flags/lithuania.svg" (click)="setLanguage('lt')" /> -->
```
Comment (ankeris) [inline src/app/@modules/topheader/topheader.component.html:9]: clean up

File: src/app/@modules/vendor/vendor-listings-page/vendor-listings.component.html [hunk 7]
```diff
@@ -1,9 +1,13 @@
...
+        <nb-card-header>
+            <app-vendor-summary-header [vendor$]="_vendor$"></app-vendor-summary-header>
+        </nb-card-header>
+        <nb-card-body>
+            <app-vendor-listings-search (onChange)="inputTextWritten($event)"></app-vendor-listings-search>
+        </nb-card-body>
+        <nb-card-footer>
+            <app-list-listings [listingsList$]="_listingsList$"></app-list-listings>
+        </nb-card-footer>
+    </nb-card>
```
Comment (ankeris) [inline src/app/@modules/vendor/vendor-listings-page/vendor-listings.component.html:12]: <3

**Pull request merged**